extensions/wikia/Search/classes/Result.php

```diff
@@ -61,7 +61,7 @@ public function getText( $field = 'text', $wordLimit = null ) {
 		$text = isset( $this->_fields[$field] ) ? $this->_fields[$field] : '';
 		$textAsString = is_array( $text ) ? implode( " ", $text ) : $text;
 
-		$textAsString = $this->limitTextLength( $textAsString, $wordLimit );
+		$textAsString = static::limitTextLength( $textAsString, $wordLimit );
 
 		// if title and description both start with the same File: prefix, remove the prefix from description
 		$textAsString = $this->removePrefix($this->findFilePrefix($this->getTitle(false)), $textAsString);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$textAsString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)